src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -349,6 +349,7 @@
     // User
     private static final String USER_ID = "userId";
     private static final String EXTERNAL_JOURNAL_LISTS = "externalJournalLists";
+    private final String USER_NAME;
```

Comment:
Since this is not a static variable, please rename it to `userName`.

Reply:
renamed